src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -99,6 +99,11 @@ class DefaultStorageKeyAllocator
     static StorageKeyName FabricNOC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/n", fabric); }
     static StorageKeyName FabricICAC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/i", fabric); }
     static StorageKeyName FabricRCAC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/r", fabric); }
+    static StorageKeyName FabricVVSC(FabricIndex fabric) { return StorageKeyName::Formatted("f/%x/vvvc", fabric); }
```

Comment:
Why vvvc, not vvsc?  Worth documenting if there is a good reason.

Reply:
It's a typo, will fix.